src/pkg/packager/deploy.go

```diff
@@ -357,6 +378,7 @@ func (p *Packager) deployComponent(ctx context.Context, component v1alpha1.ZarfC
 		healthCheckContext, cancel := context.WithTimeout(ctx, p.cfg.DeployOpts.Timeout)
 		defer cancel()
 		spinner := message.NewProgressSpinner("Running health checks")
+		l.Info("running health checks")
```

Comment:
Non-blocking: Is there a piece of data we can use to link this to the component e.g. component name? If we grow this part of the system to be concurrent and healthchecks are interleaved together, we want the ordering to not matter for the logs to make sense.

Reply:
Yup added the chart name